### PR TITLE
[Python] Match unpacking operators

### LINKED ISSUE #773: [Python] * and ** in method definitions are unscoped
https://docs.python.org/dev/tutorial/controlflow.html#keyword-arguments

    def example(test, *args, **kwargs): # <- scope my asterisks please!
        print(test)
        print(*args)
        print(list(range(*args)))
        print(*kwargs)
        blah = { 'def': 2, 'testing123': 'TEST' }
        combined = { **kwargs, **blah }
        print(combined)
    example('hello world', 3, 6, testing123='test', abc=1)

### PR DESCRIPTION
**Pull request by FichteFoll**
Follow-up on https://github.com/sublimehq/Packages/pull/743. Since this is also runs into https://github.com/SublimeTextIssues/Core/issues/1190, please confirm locally that the tests pass as I can't.

Fixes #773.

### CODE REVIEW

File: Python/Python.sublime-syntax [hunk 1]
```diff
@@ -395,6 +391,15 @@ contexts:
     - match: '@'
       scope: keyword.operator.matrix.python
 
+  allow-unpack-operators:
+    # Match unpacking operators, if present
+    - match: \*
+      scope: keyword.operator.unpacking.python
+    - match: \*\*
+      scope: keyword.operator.unpacking.python
```
Comment (keith-hall) [inline Python/Python.sublime-syntax]: This pattern will never be matched due to the single `*` pattern coming before it.

Comment (keith-hall) [inline Python/Python.sublime-syntax]: Currently, 3 or more stars in a row will be scoped as if valid, is that what we want?

Comment (keith-hall) [inline Python/Python.sublime-syntax]: As will stars with spaces between them, which I don't think is valid.

Comment (FichteFoll) [inline Python/Python.sublime-syntax]: Nice spot. All fixed.

**Comment by wbond** [comment]
Tests passed, thanks!

**Pull request merged**